### DISCUSSION TITLE
Use `@use` and `@forward` instead of `@import`

### DISCUSSION THREAD

**Discussion by sonikasharma1403** [vmware-clarity/ng-clarity#140] [Q&A]
Is there any plan of migrating to `@use` and `@forward` from `@import`?

**Comment by bbogdanov** [comment]
Hi @sonikasharma1403,

We didn't think of that. Thank you bringing our attention to it. I'll add it to the backlog for further discussion.

**Reply by sonikasharma1403** [reply]
can we have a rough idea of when we can expect this change?

**Reply by bbogdanov** [reply]
We don't have a timeline. As the team is small and we have a lot on our plate I don't believe it will be anytime soon. 

But we are open for a contribution.
